digital_image_processing/test_digital_image_processing.py

```diff
@@ -0,0 +1,67 @@
+"""
+PyTest's for Digital Image Processing
+"""
+
+# add change_contrast
+from PIL import Image
+import digital_image_processing.change_contrast as cc
+# add canny
+import digital_image_processing.edge_detection.canny as canny
+import cv2
+# add gen_gaussian
+from cv2 import imread, cvtColor, COLOR_BGR2GRAY, imshow, waitKey
+from numpy import array, pi, mgrid, exp, square, zeros, ravel, dot, uint8
+import digital_image_processing.filters.gaussian_filter as gg
+# add convolve
+import digital_image_processing.filters.convolve as conv
+# add median
+import digital_image_processing.filters.median_filter as med
+# add sobel_filter
+import digital_image_processing.filters.sobel_filter as sob
+# add img
```

Comment:
Several of these comments add no new information and can be safely eliminated.

Reply:
Removed a lot of unnecessary comments 👍